libs/BehaviorKit/source/BehaviorKit.cpp

```diff
@@ -10,6 +10,8 @@ namespace leka {
 
 using namespace std::chrono;
 
+inline constexpr auto reinforcer_duration = 2700ms;
```

Comment:
ici tu veux `static` parce que c'est juste pour cette TU. 

```suggestion
static constexpr auto kReinforcerDuration = 2700ms;
```